src/plugins/folderView/components/ListView.vue

```diff
@@ -43,6 +43,7 @@
             </thead>
             <tbody>
                 <list-item v-for="(item,index) in sortedItems"
+                    :key="index"
```

Comment:
Index is ordinal, you can't use it as a unique key anywhere because it does not uniquely identify the object in the collection. It changes depending on order. Best bet is to use a keystring here. I would amend composition-loader to add a keystring to each item [when they're created](https://github.com/nasa/openmct/blob/2b3a03e5cd675d3bb398d86ed9bede914fdfb82e/src/plugins/folderView/components/composition-loader.js#L34).

Reply:
Agreed, made changes